packages/@aws-cdk/cdk/lib/core/tokens.ts

```diff
@@ -102,11 +102,16 @@ export class Token {
 }
 
 /**
- * Returns true if obj is a token (i.e. has the resolve() method)
+ * Returns true if obj is a token (i.e. has the resolve() method or is a string
+ * that includes token markers).
  * @param obj The object to test.
  */
 export function isToken(obj: any): obj is Token {
```

Comment:
Wouldn't `hasToken()` or `containsToken()` be more appropriate?

Reply:
Renamed to `unresolved`, what do you think? (should it be `isUnresolved`)?